java/code/src/com/suse/manager/matcher/MatcherJsonIO.java

```diff
@@ -158,12 +158,22 @@ public MatcherJsonIO() {
      * about systems on this Server
      */
     public List<SystemJson> getJsonSystems(boolean includeSelf, String arch, boolean selfMonitoringEnabled) {
+        List<String> vCoreCountedChannelFamilies = of("MICROOS-ARM64", "MICROOS-X86", "MICROOS-Z", "MICROOS-PPC")
+                .flatMap(cf -> of("", "-ALPHA", "-BETA").map(s -> cf + s))
+                .collect(toList());
+
         Stream<SystemJson> systems = ServerFactory.list(true, true).stream()
             .map(system -> {
                 Long cpus = system.getCpu() == null ? null : system.getCpu().getNrsocket();
                 Set<String> entitlements = system.getEntitlementLabels();
                 boolean virtualHost = entitlements.contains(EntitlementManager.VIRTUALIZATION_ENTITLED) ||
                         !system.getGuests().isEmpty();
+                boolean countVCores = !virtualHost && system.getInstalledProductSet()
+                        .map(s -> s.getBaseProduct().getChannelFamily()).stream()
+                        .anyMatch(cf -> vCoreCountedChannelFamilies.contains(cf.getLabel()));
+                if (countVCores) {
```

Comment:
I think it would be a good idea to add a comment to explain that this is an hack and the proper solution requires adding a new json field and modifying subscription matcher, but we do not consider the effort worth at the moment.